src/video/SDL_video.c

```diff
@@ -2141,7 +2141,7 @@ void SDL_SetWindowPosition(SDL_Window *window, int x, int y)
         }
     }
 
-    if ((window->flags & SDL_WINDOW_FULLSCREEN) != 0) {
+    if ((window->flags & SDL_WINDOW_FULLSCREEN)) {
```

Comment:
Extra parens can be removed

Reply:
I think this one would trigger a warning !

Reply:
Removing it would trigger a warning? How??

Reply:
no, you're right

Reply:
I thought about boolean operator warning